Tests/RevenueCatUITests/Templates/Template3ViewTests.swift

```diff
@@ -15,6 +15,14 @@ class Template3ViewTests: BaseSnapshotTest {
         .snapshot(size: Self.fullScreenSize)
     }
 
+    func testTabletPaywall() {
+        PaywallView(offering: Self.offering.withLocalImages,
+                    introEligibility: Self.eligibleChecker,
+                    purchaseHandler: Self.purchaseHandler)
+        .environment(\.userInterfaceIdiom, .pad)
```

Comment:
This is one of the main reasons for creating this: we can inject a custom value for snapshot tests.